mesos-docker/run/run.sh

```diff
@@ -54,17 +53,49 @@ function docker_ip {
 }
 
 function print_host_ip {
+  printMsg "IP address of the docker host machine is $(get_host_ip)"
+}
+
+function get_host_ip {
   if [[ "$(uname)" == "Darwin"  ]]; then
     #Getting the IP address of the host as it seen by docker container
     masterContainerId=$(docker ps -a | grep $MASTER_CONTAINER_NAME | awk '{print $1}')
-    hostIpAddr=$(docker exec -it $masterContainerId /bin/sh -c "sudo ip route" | awk '/default/ { print $3 }')
+    docker exec -it $masterContainerId /bin/sh -c "sudo ip route" | awk '/default/ { print $3 }'
+  else
+    docker_ip
+  fi
+}
 
-    printMsg "The IP address of the host inside docker $hostIpAddr"
+function default_mesos_lib {
+  if [[ "$(uname)" == "Darwin"  ]]; then
+    echo "/usr/local/lib/libmesos.dylib"
   else
-    printMsg "The IP address of the host inside docker $(docker_ip)"
+    echo "/usr/local/lib/libmesos.so"
```

Comment:
It can also be in `/usr/lib/libmesos.so`. I think the simplest solution would be to test the different possible locations, and pick the one that exists.